pkg/services/sqlstore/org.go

```diff
@@ -241,6 +241,16 @@ func DeleteOrg(cmd *models.DeleteOrgCommand) error {
 			"DELETE FROM org_user WHERE org_id = ?",
 			"DELETE FROM org WHERE id = ?",
 			"DELETE FROM temp_user WHERE org_id = ?",
+			"DELETE FROM ngalert_configuration WHERE org_id = ?",
+			"DELETE FROM alert_configuration WHERE org_id = ?",
+			"DELETE FROM alert_instance WHERE rule_org_id = ?",
+			"DELETE FROM alert_notification WHERE org_id = ?",
+			"DELETE FROM alert_notification_state WHERE org_id = ?",
+			"DELETE FROM alert_rule WHERE org_id = ?",
+			"DELETE FROM alert_rule_tag WHERE EXISTS (SELECT 1 FROM alert WHERE alert.org_id = ? AND alert.id = alert_rule_tag.alert_id)",
+			"DELETE FROM alert_rule_version WHERE rule_org_id = ?",
+			"DELETE FROM alert WHERE org_id = ?",
+			"DELETE FROM annotation WHERE org_id = ?",
```

Comment:
I think that what @yuri-tceretian suggests seems sensible, however, I'm not sure that's what we want to do as part of this PR.

Using (or creating) all of the "delete commands" for each entity within Grafana seems like a widespread effort. For example, I'm thinking that involves:
- Going through each entity's delete command (and creating one if it doesn't exist)
- Extract out any logic that we don't care about as part of this action to avoid any unwanted actions/steps
- Wrap everything within a transaction (and make sure it works as expected on every engine)
- Time the duration on the worst-case scenarios to decide whether we should go with a sync deletion (keep the user waiting) or an async deletion (which would bring an extra set of problems). 

Generally, when we're talking about tasks that require such an effort we tend to run them by the @marefr and his squad to make sure we're on the same page. 

I think that what we're doing as part of this PR is an incremental improvement to what we currently have. Even if it's far from perfect, it's still better than what we currently have right now and simple enough that allows us to tear it 
down easily once the best thing comes along.